tests/apidocs/endpoints/projects/test-user-feedback.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from django.test.client import RequestFactory
+from django.utils import timezone
+
+from tests.apidocs.util import APIDocsTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+
+
+class ProjectUserFeedbackDocs(APIDocsTestCase):
+    def setUp(self):
+        organization = self.create_organization()
+        project = self.create_project(name="foo", organization=organization, teams=[])
+        event = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "message": "oh no",
+                "timestamp": iso_format(before_now(seconds=1)),
+            },
+            project_id=project.id,
+        )
+        group = self.create_group(project=project, message="Foo bar")
+        self.create_userreport(
+            date_added=timezone.now(), group=group, project=project, event_id=event.event_id,
+        )
+
+        self.url = u"/api/0/projects/{}/{}/user-feedback/".format(organization.slug, project.slug)
```

Comment:
this one doesn't use `reverse` like the others because it can't resolve the `|` from https://github.com/getsentry/sentry/blob/master/src/sentry/api/urls.py#L1507